test/runtime/ct.go

```diff
@@ -115,7 +104,35 @@ var _ = Describe("RuntimeConntrackTable", func() {
 		}
 	}
 
-	// containersMeta retuns a map where the key is the container name and the
+	initialize := func() {
+		logger = log.WithFields(logrus.Fields{"testName": "RuntimeConntrack"})
+		logger.Info("Starting")
+		vm = helpers.CreateNewRuntimeHelper(helpers.Runtime, logger)
+		err := vm.WaitUntilReady(100)
+		Expect(err).To(BeNil())
+		vm.NetworkCreate(helpers.CiliumDockerNetwork, "")
+	}
+
+	BeforeEach(func() {
+		once.Do(initialize)
+
+		res := vm.SetPolicyEnforcement(helpers.PolicyEnforcementAlways)
```

Comment:
Why do we set policy to `always` before each test and `default` after each test? I think we are not testing scenarios with policy `default` at all this way?

Reply:
The policy is set to `always` on all tests but it's only set to `default` on a couple tests